common/src/main/scala/explore/components/Tile.scala

```diff
@@ -45,11 +53,16 @@ object Tile {
          (1024                           -> TileLGW)
     )
 
+  def renderPortal(mountNode: Option[raw.ReactDOM.Container], info: VdomNode): VdomNode =
+    mountNode.map(node => ReactPortal(info, node))
```

Comment:
Is ReactPortak defined in scalajs-react?

Reply:
Yes